.circleci/config.yml

```diff
@@ -122,7 +122,7 @@ commands:
       - run:
           name: "Build Benchmarks - << parameters.benchmark_class >>"
           command: |
-            make benchmarks-basic-build NUM_THREADS=16 MAX_HIGH_MEM_JOBS=8 MAX_LINK_JOBS=8
+            make benchmarks-basic-build NUM_THREADS=8 MAX_HIGH_MEM_JOBS=8 MAX_LINK_JOBS=8
```

Comment:
xlarge also has 32G vs the  previous 64G maybe MAX_HIGH_MEM_JOBS also needs to be reduced? (I have no idea what the requirements there are so if you think it's fine than it can stay like this)

Reply:
If nothing breaks I am inclined to let it stay as is :) ..